llvm/test/CodeGen/RISCV/addimm-mulimm.ll

```diff
@@ -879,12 +880,31 @@ define i64 @mulneg3000_sub8990_c(i64 %x) {
 define i1 @pr53831(i32 %x) {
 ; RV32IMB-LABEL: pr53831:
 ; RV32IMB:       # %bb.0:
-; RV32IMB-NEXT:    li a0, 0
+; RV32IMB-NEXT:    addi a1, a0, 1
```

Comment:
This test is checking for an infinite combine loop, but this result is the missed optimization I mentioned in the review summary.